hydra-access-controls/lib/hydra/config.rb

```diff
@@ -11,6 +11,8 @@ def []= key, value
           self.permissions = value
         when :user_model
           self.user_model = value
+        when :id_to_resource_uri
```

Comment:
This is for the case when the URI you want to display is -not- the fedora URI, whereas #translate_id_to_uri is used for persistence to LDP and thus must be the Fedora URI.